halyard-cli/src/main/java/com/netflix/spinnaker/halyard/cli/command/v1/config/providers/kubernetes/KubernetesEditAccountCommand.java

```diff
@@ -183,6 +227,18 @@ protected Account editAccount(KubernetesAccount account) {
       throw new IllegalArgumentException("Set either --omit-namespace or --[add/remove]-omit-namespace");
     }
 
+    try {
+      account.setKinds(updateStringList(account.getKinds(), kinds, addKind, removeKind));
+    } catch (IllegalArgumentException e) {
+      throw new IllegalArgumentException("Set either --kind or --[add/remove]-kind");
```

Comment:
"or use --[add/remove]-kind" maybe.

It's not clear to me when this exception would be thrown. Perhaps the error could be explained before offering a solution.

Reply:
We can address this in a another PR, since this pattern is all over the place. What the user winds up seeing is

```
$ hal config provider kubernetes account edit my-account --kinds ReplicaSet --remove-kind Secret

ERROR Set either --kinds or --[add/remove]-kind
```

I think it's acceptable for now but we can improve the wording later if you prefer.